ApiDoctor.Validation/Error/IssueLogger.cs

```diff
@@ -195,12 +195,35 @@ private bool HandleSuppression(ValidationError error)
                 }
                 else
                 {
-                    issues.Add((T)issue);
+                    if (DocSet.SchemaConfig != null && DocSet.SchemaConfig.TreatErrorsAsWarningsWorkloads.Any(s => !string.IsNullOrWhiteSpace(this.Source) && this.Source.Contains(s)))
+                    {
+                        if (!issue.IsWarning)
+                        {
+                            this.warnings.Add(new ValidationWarning(issue.Code, issue.Source,
+                                $"Treating Error as Warning {Environment.NewLine} {issue.Message}"));
+                        }
+                    }
+                    else
+                    {
+                        issues.Add((T)issue);
+                    }
                 }
             }
             else
             {
-                issues.Add((T)issue);
+
+                if (DocSet.SchemaConfig != null && DocSet.SchemaConfig.TreatErrorsAsWarningsWorkloads.Any(s => !string.IsNullOrWhiteSpace(this.Source) && this.Source.Contains(s)))
```

Comment:
do a case-insensitive string compare on the 'contains()'

Reply:
I doubt this particular overload is availble in .NET 4.5 which we are currently using. I am using the IContains extension method.